godtools/App/Features/AppLanguage/Presentation/DownloadableLanguages/ViewModel/DownloadableLanguagesViewModel.swift

```diff
@@ -42,6 +43,12 @@ class DownloadableLanguagesViewModel: ObservableObject {
         self.downloadToolLanguageUseCase = downloadToolLanguageUseCase
         self.removeDownloadedToolLanguageUseCase = removeDownloadedToolLanguageUseCase
         
+        DownloadableLanguagesViewModel.languageDownloadHandler.setDelegate(self)
+        DownloadableLanguagesViewModel.languageDownloadHandler
```

Comment:
Trying to remember where I saw this, but you can also access class level variables using keyword ```Self``` capitalized "s".

Reply:
Oh cool!